### ISSUE TITLE
Crafting recipe for minifreezer/minifridge/washing machine

### ISSUE THREAD

**Issue by KorGgenT** [KorGgenT/Cataclysm-DDA#3]
Minifreezer/Minifridge appear to use the same parts, just with different settings.

https://www.doityourself.com/stry/troubleshooting-a-chest-freezer-that-runs-continuously
https://yourairexperts.com/article/what-are-evaporator-and-condenser-coils-and-how-do-they-help-cool-your-home
https://www.air-conditioning-and-refrigeration.com/Refrigerator-components.html

chest freezer:
condensor coil
evaporator coil
compressor
relay
thermostat
Refridgerant

Evaporator Coil:
made of copper, steel, or aluminum tubes and panels

Condensor Coil:
tubes, fins, compressor, fan, copper tubing, valbes, switches

Compressor:
basically a small electric motor
